.github/workflows/ci.yml

```diff
@@ -83,14 +85,13 @@ jobs:
     steps:
       - name: checkout_repository
         uses: actions/checkout@v2.4.0
-        with:
-          fetch-depth: 1
       - name: install_rust
-        uses: hecrj/setup-rust-action@v1
```

Comment:
Let's split these CI workflow changes to another PR; I've used both setup actions, and both work fine. Altough, the nice thing of the actions-rs one is that it allows using a minimal profile, (not sure about Hecrj's action), and is probably more actively maintained (but again, not sure here). Taking these things into account, I'm not sure whether it's a change I would want to make, but I have nothing against it either.